docs/source/conf.py

```
@@ -373,6 +373,50 @@ def skip_pydantic_special_attrs(app: object,
 
         return skip
 
+    def clean_markdown_from_docstrings(app, docname, source):
```

Ruff (0.14.10)

Unused function argument: `app`

(ARG001)